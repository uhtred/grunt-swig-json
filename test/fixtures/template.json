{
    "pagename": "Awesome people",
    "authors": [
    	"Paul",
    	"Jim",
    	"Jane"
    ]
}