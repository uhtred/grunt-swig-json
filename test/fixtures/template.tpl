<h1>{{ pagename|title }}</h1>
<ul>
	{% for author in authors %}
		{% include "./includes/item.tpl" %}
	{% endfor %}

	{% include "./includes/item.tpl" { "author": "Daniel" } %}
</ul>