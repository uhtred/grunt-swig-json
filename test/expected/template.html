<h1>Awesome People</h1>
<ul>

    <li class="first">Paul</li>

    <li>Jim</li>

    <li>Jane</li>

</ul>